### PR TITLE
[16.0] [OU-ADD] account_edi_ubl: Nothing to do

### PR DESCRIPTION
**Pull request by NgoHoangHung**

### CODE REVIEW

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| account_edi_ubl_bis3                      |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| account_edi_ubl_cii                             |                      |                                                 |
+| account_edi_ubl_cii                             |Done                  |Merged into account_edi_ubl_cii                  |
```
Comment (hoangtiendung070797) [inline docsource/modules150-160.rst]: @NgoHoangHung I see that you are confusing something here. Please check again

Comment (NgoHoangHung) [reply docsource/modules150-160.rst]: sorry, this is huge my mistake.

**Comment by hoangtiendung070797** [comment]
@NgoHoangHung I see the account_edi_ubl_cii module has an `upgrade_analysis.txt` file. I think you need to handle it

**Pull request closed without merging**